### PR TITLE
Make HTTP cache dir configurable in INI file

### PR DESCRIPTION
**Pull request by ronnix**
Most settings are in the INI file, but for some reason this one was using an environment variable.

(Also, it makes it easier to solve a pain point where in development I want to prevent Zam commands run in a container from creating files owned by root on the host.)

### CODE REVIEW

File: repondeur/zam_repondeur/services/fetch/http.py [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
+    Called automatically via config.include("zam_repondeur.services.fetch.http")
+    """
+    session = requests.session()
+    http_cache_dir = config.registry.settings["zam.http_cache_dir"]
+    cached_session = CacheControl(session, cache=FileCache(http_cache_dir))
+    config.registry.registerUtility(component=cached_session, provided=IHTTPSession)
 
-cached_session = CacheControl(session, cache=FileCache(HTTP_CACHE_DIR))
+
+def get_http_session(registry: Optional[Registry] = None) -> CacheControl:
```
Comment (davidbgk) [inline repondeur/zam_repondeur/services/fetch/http.py]: Do we use that optional param somewhere?

File: repondeur/zam_repondeur/services/fetch/http.py [hunk 2]
```diff
@@ -1,16 +1,29 @@
...
+    if registry is None:
+        registry = get_current_registry()
+    return registry.queryUtility(IHTTPSession)
```
Comment (davidbgk) [inline repondeur/zam_repondeur/services/fetch/http.py]: What if we add `cached_session = get_http_session()` at the end of the file? I guess we don't want to instanciate it this early?

File: repondeur/zam_repondeur/services/fetch/http.py (hunk 1 above)
Comment (ronnix) [reply repondeur/zam_repondeur/services/fetch/http.py]: Not yet, but I feel less bad if I give the caller the opportunity of passing it as a param and avoiding a `get_current_registry()` call when not necessary...

(Similar to https://github.com/betagouv/zam/blob/master/repondeur/zam_repondeur/templating.py#L43-L45)

File: repondeur/zam_repondeur/services/fetch/http.py (hunk 2 above)
Comment (ronnix) [reply repondeur/zam_repondeur/services/fetch/http.py]: Exactly, we don't want to initialize it at import time (too early). We want to wait for the app to read its config and call the `includeme()` for initialization.

File: repondeur/zam_repondeur/services/fetch/http.py (hunk 1 above)
Comment (davidbgk) [reply repondeur/zam_repondeur/services/fetch/http.py]: Got it 👍

File: repondeur/zam_repondeur/services/fetch/http.py (hunk 2 above)
Comment (davidbgk) [reply repondeur/zam_repondeur/services/fetch/http.py]: OK

**Pull request merged**